lib/swing-lib-test-support/src/main/java/org/triplea/ui/events/queue/test/support/EventQueueAssertions.java

```diff
@@ -0,0 +1,47 @@
+package org.triplea.ui.events.queue.test.support;
+
+import com.google.common.base.Preconditions;
+import org.hamcrest.Matcher;
+import org.triplea.test.common.CustomMatcher;
+import org.triplea.ui.events.queue.ViewData;
+
+/**
+ * Assertions that can be used with 'EventQueueTestSupport'. A typical test will set up the queue
+ * test support, an event queue and controller. The queue test support will be registered as a view
+ * with the event queue. Then, the test will publish a view event to the event queue. Test support
+ * is then used to verify any response events placed back on the queue by the controller.
+ */
+public class EventQueueAssertions {
+
+  /**
+   * Use to assert that the most recently received controller event had a specific event type. EG:
+   * <code>
+   *   <pre>
+   * assertThat(eventQueueTestSupport, lastControllerEventTypeIs(ControllerEvents.EVENT_TYPE));
+   *   </pre>
+   * </code>
+   */
```

Comment:
TODO: out of date comment

(keep in mind the test-support class has most of what would need to be here)